coderd/audit.go

```diff
@@ -185,12 +190,23 @@ func (api *API) convertAuditLog(ctx context.Context, dblog database.GetAuditLogs
 		}
 	}
 
-	isDeleted := api.auditLogIsResourceDeleted(ctx, dblog)
-	var resourceLink string
+	var (
+		additionalFieldsBytes = []byte(dblog.AdditionalFields)
+		additionalFields      AdditionalFields
+		err                   = json.Unmarshal(additionalFieldsBytes, &additionalFields)
+	)
+	if err != nil {
+		api.Logger.Error(ctx, "unmarshal additional fields", slog.Error(err))
+	}
```

Comment:
If this happens, should we try to populate `additionalFields` with something to show it failed, maybe like `unknown`.